Port in-product tile instructions for some OAuth integrations to the README that gets parsed inside the "Configuration instructions" section (#23673)
DataDog/integrations-core: klaviyo/README.md

## Setup

> The user who authenticates this integration must have access to the following Klaviyo Scopes:

Reviewer: ```suggestion
To authenticate this integration, you must have access to the following Klaviyo Scopes:
```

- `accounts:read`
- `events:read`
- `flows:read`
- `metrics:read`

> To add a new Klaviyo account, click the **Authorize** button and follow the instructions. Once you are redirected back to this page and the authentication is successful, your logs should be available within 5 minutes.

Reviewer: ```suggestion
To add a Klaviyo account, click **Authorize** and follow the instructions. After you are redirected back to this page and authentication succeeds, your logs should be available within 5 minutes.
```

#### Install Datadog Integration in Klaviyo
Within your Klaviyo account, first add the Datadog integration. The integration allows Datadog
to see Klaviyo events and metrics via the Klaviyo API.
To add a new Klaviyo account, click the **Authorize** button and follow the instructions. Once you are redirected back to this page and the authentication is successful, your logs should be available within 5 minutes.

1. Log in to your [Klaviyo account][2].
2. In the left-side panel, navigate to **Integrations**.
3. Click **Add integrations**.
4. Search for Datadog and click on the tile.
5. Click **Install**. 

> 6. Navigate to Datadog, then log in.

Author: I understand this section is now obsolete, but correct me if I'm wrong.

Need help? Contact [Datadog support][3].

[1]: https://www.klaviyo.com/
[2]: /logs?query=source%3Aklaviyo%2A

> [3]: /logs/pipelines/indexes

Author: Oof good catch, fixed!
